app/views/sessions/new.html.erb

```diff
@@ -33,15 +33,12 @@
 
     <%= f.text_field :username, :label => t(".email or username"), :autofocus => true, :tabindex => 1, :value => params[:username] %>
 
-    <div class="row">
-      <div class="col">
-        <%= f.label :password, t(".password"), :class => "form-label" %>
-      </div>
-      <div class="col text-end">
-        <small><%= link_to(t(".lost password link"), user_forgot_password_path) %></small>
-      </div>
+    <div class="d-flex flex-wrap column-gap-3 justify-content-between align-items-baseline">
+      <%= f.label :password, t(".password"), :class => "my-2" %>
```

Comment:
https://github.com/openstreetmap/openstreetmap-website/pull/4935#discussion_r1661380070 talks about the height of the whole row, yet the margin is applied just to the label. Why not:
```suggestion
    <div class="d-flex flex-wrap column-gap-3 justify-content-between align-items-baseline mb-2">
      <%= f.label :password, t(".password") %>
```

Before/after:
![image](https://github.com/openstreetmap/openstreetmap-website/assets/4158490/881eed15-8ce8-4e8b-b85e-5838203b6681) ![image](https://github.com/openstreetmap/openstreetmap-website/assets/4158490/c563252d-4428-483f-8dc2-abe7bcaa1f51)

Also we usually add bottom margin, not both top and bottom.